skillmap/src/components/AppModal.tsx

```diff
@@ -122,7 +123,8 @@ export class AppModalImpl extends React.Component<AppModalProps, AppModalState>
             { label: lf("CANCEL"), onClick: this.handleOnClose },
             { label: lf("RESTART"), onClick: () => {
                 tickEvent("skillmap.activity.restart", { path: mapId, activity: activity!.activityId });
-                dispatchRestartActivity(mapId, activity!.activityId);
+                if (showCodeCarryoverModal) dispatchShowCarryoverModal(mapId, activity!.activityId);
```

Comment:
is it weird to have two subsequent modals like this? do the modals look different enough that kids won't just rapidly click on the rightmost button twice in a row?

Reply:
I think we should maybe take a look at it in a design meeting; could be confusing.